### PR TITLE
Updated documentation to build JS SDK

### PR DESCRIPTION
**Pull request by ihorserba**

### CODE REVIEW

File: sdks/js/README.md [hunk 1]
```diff
@@ -5,48 +5,38 @@ See examples in `examples` folder and documentation in `/docs/js` folder
...
 ```
 
-##### Follow installation instructions in README.md file ([Building from sources](https://github.com/megamk/opus-recorder#building-from-sources) section) to build from source
+### Rebuilding the recorder (optional)
 
-##### Rebuild recorder
-```bash
-npm run make recorder
-```
+* Note: No need rebuilding the recorder if it was not changed.
```
Comment (vocoded) [inline sdks/js/README.md:32]:
```suggestion
* Note: No need to rebuild the recorder if it was not changed.
```

**Review by vocoded — COMMENTED** [review]
Does this work on all platforms or just macOS?

File: sdks/js/README.md (hunk 1 above)
Comment (ihorserba) [reply sdks/js/README.md]: Changed

**Comment by ihorserba** [comment]
> Does this work on all platforms or just macOS?

Verified on the following platforms:

Ubuntu 20.04.3 LTS
MacOS 12.0.1
Windows 10 Pro x64

**Review by megamk — APPROVED** [review]

**Pull request merged**